viewer/src/main/webapp/viewer-html/components/Edit.js

```diff
@@ -1724,5 +1745,7 @@ Ext.define("viewer.components.Edit", {
             }
         }
         return map;
+    },
+    onFormFieldChange: function (me, input, newValue, oldValue) {
```

Comment:
Remove the me argument, since its part of a class it should be obvious what the scope is